### DISCUSSION TITLE
Import/Export List Builders as CSV

### DISCUSSION THREAD

**Discussion by YozoraXCII** [linaspurinis/mdblist.com#4] [Ideas]
Maybe more long-term, but it would be neat if we could export and import the builders/templates for our lists in CSV or similar - I currently have over 100 lists and I'm contemplating updating a majority of them, which currently is a manual list-by-list process.

**Comment by linaspurinis** [comment]
Could you explain in more detail? If you want to save your list search terms, you can:

1.  Save it as a filter
2. Copy and save the list URL. Go to your lists, click edit Movie Filter, click search, and copy the URL. This URL will have all parameters for a list and you will be able to recreate list from this single URL.

**Reply by YozoraXCII** [reply]
Thanks for the reponse - Wasn't aware that could be done and will definitely make life a little bit easier.

I think people are quite common with being able to import/export things to CSV format (Trakt lets you do it for lists for example.

My thinking was that I could export my lists as CSV format - this would let me see and edit the exact attributes that produce my lists.

For example - I have 50+ lists that I would like to batch update the sort method on, or potentially add a limit to. Your copying URL method would work, but would be very tedious and I'd have to meticulously track which ones I'd done. If I have a CSV file I can open it in excel and batch edit what I need in there, then re-import the file so that my lists are updated.

FWIW this got a +1 on the Discord channel, so I'm not the sole person that would like this

**Reply by linaspurinis** [reply]
Trakt allows you to export the content of the list, you can do the same with mdblist in json. Also, you can use online tool to convert URL parameters to json, for example:
https://www.convertonline.io/convert/query-string-to-json
Just paste the URL parameters and you can modify them, and convert them back to the query string.
